docs/jwst/pathloss/reference_files.rst

```diff
@@ -52,3 +56,80 @@ as the behavious or each slice is considered identical.
 The NIRSPEC MSASPEC reference file has 2 sets of 4 extensions, one for the 1x1
 aperture size, and one for the 1x3 aperture size.  Currently there are no other
 aperture sizes.
+
+The NIRISS SOSS reference file has 1 extension in addition to the primary
+header unit.  It contains a 3-dimensional array of float32 correction values.
+The dimensions of the array are 1x2048x17.  The first dimension is a dummy to
```

Comment:
Earlier in the docs it quoted dimensions of 1x204**0**x17. Which is it?

Reply:
The data array in the reference file is 1x2040x17.  But the pathloss correction vector written to the science datamodel has length 2048 (it's padded with 1's at the ends).